tools/v8-prof/tick-processor.js

```diff
@@ -0,0 +1,67 @@
+// Copyright 2012 the V8 project authors. All rights reserved.
```

Comment:
I originally wasn't sure since I transliterated this from the bash script which was V8's. This is original code though. I've removed the copyright.